enhancements/installer/cluster-api-based-installations.md

```diff
@@ -0,0 +1,347 @@
+(conversion, admission, validation), and managing the lifecycle of Custom Resource Definitions.
+
+Over time, `envtest` matured in a way that now can be used to run controllers in a local environment,
+reducing or eliminating the need for a full Kubernetes cluster to run controllers.
+
+At a high level, the local control plane is responsible for:
+- Setting up certificates for the apiserver and etcd.
+- Running (and cleaning up, on shutdown) the local control plane components.
+- Installing any required component, like Custom Resource Definitions (CRDs)
+    - For Cluster API core the CRDs are stored in `data/data/cluster-api/core-components.yaml`.
+    - Infrastructure providers are expected to store their components in `data/data/cluster-api/<name>-infrastructure-components.yaml`
+- Upon install, the local control plane takes care of modifying any webhook (conversion, admission, validation) to point to the `host:post` combination assigned.
+    - Each controller manager will have its own `host:port` combination assigned.
+    - Certificates are generated and injected in the server, and the client certs in the api-server webhook configuration.
+- For each process that the local control plane manages, a health check (ping to `/healthz`) is required to pass similarly how, when running in a Deployment, a health probe is configured.
+
+#### Manifests
+
+The Installer will produce the CAPI manifests as part of the `manifests` target, writing them to a new
+`cluster-api` directory alongside the existing `manifests` and `openshift` directories:
+
+```shell=
+$ ./openshift-install create manifests --dir install-dir
+INFO Credentials loaded from the "default" profile in file "~/.aws/credentials" 
+INFO Consuming Install Config from target directory 
+INFO Manifests created in: install-dir/cluster-api, install-dir/manifests and install-dir/openshift 
+$ tree install-dir/cluster-api/
+install-dir/cluster-api/
+├── 00_capi-namespace.yaml
+├── 01_aws-cluster-controller-identity-default.yaml
+├── 01_capi-cluster.yaml
+├── 02_infra-cluster.yaml
+├── 10_inframachine_mycluster-6lxqp-master-0.yaml
+├── 10_inframachine_mycluster-6lxqp-master-1.yaml
+├── 10_inframachine_mycluster-6lxqp-master-2.yaml
+├── 10_inframachine_mycluster-6lxqp-master-bootstrap.yaml
+├── 10_machine_mycluster-6lxqp-master-0.yaml
+├── 10_machine_mycluster-6lxqp-master-1.yaml
+├── 10_machine_mycluster-6lxqp-master-2.yaml
+└── 10_machine_mycluster-6lxqp-master-bootstrap.yaml
+
+1 directory, 12 files
+```
+
+The manifests within this `cluster-api` directory will not be written to the cluster or included in bootstrap ignition.
+In future work, we expect these manifests to be pivoted to the cluster to enable the target cluster to take over managing
+its own infrastructure.
+
+The Cluster API manifests will be generated with `.spec` fields in the [manifest asset target][asset] with `.status`
```

Comment:
What is going to update the status values in the manifests?

Reply:
Have the same question, assuming that the manifests have already been generated, we don't overwrite them anymore

Reply:
Is this statement about status referring to the local control plane or are we talking about once we have pivoted, not clear since the previous paragraph talks about pivoting.
I expect the answer to differ based on which of these we are in

Reply:
Clarified the wording to indicate that this is referring to the fact that the installer will serialize the manifests to the installer host after install